app/src/main/res/values/colors.xml

```diff
@@ -1,6 +1,18 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <color name="colorPrimary">#008577</color>
-    <color name="colorPrimaryDark">#00574B</color>
-    <color name="colorAccent">#D81B60</color>
+
+    <color name="darkGrey">#871812</color>
```

Comment:
is it mandatory to do this?

Reply:
I forgot to delete it. I realized that I don't really need a User class, because I can get everything that I need from the database. Am I wrong about this?